dev.zed.Zed.yaml

```diff
@@ -51,6 +51,56 @@ modules:
         sha256: bad10f3c553a0e1854649ab59c5b2434da22ca1a54ae6138f1f53961567e1ab7
         url: https://download.gnome.org/sources/gcr/3.41/gcr-3.41.2.tar.xz
 
+  - name: libmd
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://libbsd.freedesktop.org/releases/libmd-1.1.0.tar.xz
+        sha256: 1bd6aa42275313af3141c7cf2e5b964e8b1fd488025caf2f971f43b00776b332
+        x-checker-data:
+          type: anitya
+          project-id: 15525
+          stable-only: true
+          url-template: https://libbsd.freedesktop.org/releases/libmd-$version.tar.xz
+    cleanup:
+      - /lib/pkgconfig
+      - /share/man
+
+  - name: libbsd
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://libbsd.freedesktop.org/releases/libbsd-0.12.2.tar.xz
+        sha256: b88cc9163d0c652aaf39a99991d974ddba1c3a9711db8f1b5838af2a14731014
+        x-checker-data:
+          type: anitya
+          project-id: 1567
+          stable-only: true
+          url-template: https://libbsd.freedesktop.org/releases/libbsd-$version.tar.xz
+    cleanup:
+      - /lib/pkgconfig
+      - /share/man
```

Comment:
None of these packages are part of the freedesktop SDK?

Reply:
AFAIK neither `libbsd` nor `libmd` is in the SDK, they are also compiled in other Flatpaks that need them such as [com.valvesoftware.Steam](https://github.com/flathub/com.valvesoftware.Steam/blob/f17483802f4c9411718eadee8503040c9b772263/modules.yml#L120-L140).